README.md

```diff
@@ -1,13 +1,27 @@
-PoC of analyzers which prevent usage of NSubstitute setup methods for non-virtual members.
+Analyzers to detect possible NSubstitute usage problems, such as attempts to setup non-virtual members.
 
 | Branch        | Build           | Coverage  |
 | ------------- |-------------| -----|
 | Master      | [![Build status](https://ci.appveyor.com/api/projects/status/t9lhmp61nuy3a7k5/branch/master?svg=true)](https://ci.appveyor.com/project/NSubstitute/nsubstitute-analyzers/branch/master)      |   [![Coverage Status](https://coveralls.io/repos/github/nsubstitute/NSubstitute.Analyzers/badge.svg?branch=master)](https://coveralls.io/github/nsubstitute/NSubstitute.Analyzers?branch=master) |
 | Dev      | [![Build status](https://ci.appveyor.com/api/projects/status/t9lhmp61nuy3a7k5/branch/dev?svg=true)](https://ci.appveyor.com/project/NSubstitute/nsubstitute-analyzers/branch/dev)    |   [![Coverage Status](https://coveralls.io/repos/github/nsubstitute/NSubstitute.Analyzers/badge.svg?branch=dev)](https://coveralls.io/github/nsubstitute/NSubstitute.Analyzers?branch=dev) |
 
+## Motivation
 
-C# example
-![C#](https://i.imgur.com/BK06j1U.png)
-Visual Basic example
-![VisualBasic](https://i.imgur.com/SS9ZJds.png)
-Original NSubstitute issue - [click](https://github.com/nsubstitute/NSubstitute/issues/328)
+[NSubstitute](https://github.com/nsubstitute/NSubstitute) was designed with the aim of having concise, friendly syntax for mocking. The downside of this syntax is that certain failure modes are hard to detect. One such example is attempting to mock virtual members -- NSubstitute can not see these calls so can not communicate problems such as `sub.Received().NonVirtualCall()`. Thanks to the goodness of Roslyn analyzers, we now have the option to detect cases like these (as originally proposed in [NSubstitute issue #328](https://github.com/nsubstitute/NSubstitute/issues/328)).
```

Comment:
> One such example is attempting to mock **virtual** members

I think you meant **non-virtual** members

Reply:
Oops! 😊 Thanks!